CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### Features
+
+- Add support for React Native mixed stacktraces ([#3201](https://github.com/getsentry/sentry-react-native/pull/3201))
+
+  In the current `react-native@nightly` JS errors from native modules can contain native JVM or Objective-C exception stack trace.
```

Comment:
Does the `nightly` version have a preview number or something? or this is going to be outdated very soon.

Reply:
Yes, they have a datatime timestamp, but since I assume the feature is going to stay and move to release, I didn't want to fix it on some old nightly build.

Reply:
if someone is using an older `nightly`, it won't work, maybe put between `()` the timestamp?

Reply:
Okay, I see, I will add it.